js/script2.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+var output = document.getElementById('output');
+var outputinfo = document.getElementById('outputinfo');
+var button = document.getElementById('button');
+var button2 = document.getElementById('button2');
+
+output.innerHTML = 'Wcisnij guzik jeśli chcesz zamieić stopnie' + '<br><br>'; 
+
+var degreesC;
+var degreesF;
+
+button.addEventListener('click', function()
+{	
+    degreesC = window.prompt('Prosze podać stopnie celcjusza');
+    var valueC = parseInt(degreesC);
+
+    var degreesF = degreesC * 1.8 + 32;  
+
+    if(!isNaN(valueC)) {
+            output.innerHTML = 'Stopnie Celsjusza: ' + degreesC + '&#x2103 ' + '<br>' +
+            'Stopnie Fahrenheita: ' + degreesF + '&#x2109' + '<br>'; 
+    }
+    else if (isNaN(valueC)) {
+            output.innerHTML = 'Nie wpisałeś cyfry';
+    }
+}); 
+
+button.addEventListener('click', function()
+{
+    var outputinfo = document.getElementById('outputinfo');
+    var valueC = parseInt(degreesC);
+    if(valueC == 0) {
```

Comment:
1) spacja po `if` tu i w reszcie miejsc
2) porównuj przez `===` najlepiej, `==` w wyjątkowych przypadkach i tylko z pełną świadomością po co to robisz